python/Quickstart.MD

```diff
@@ -5,130 +5,107 @@ This quickstart covers:
 
-### Create Payload File
 
-The APT-SCWM API uses JSON for passing data. The minimum required data to submit a direct supplier incident is a summary of the incident and an impact priority level.  An example of the minimum required payload:
+### Request Body  
+
+For this example, we create the request body first.  In its simplest form, the request body contains a header and a payload and conforms to this format:
 ```
 {
-"payload": {
-    "aptBusinessObjectSummary": "Automation test from samples folder.",
-     "directSupplierImpact": {
-        "businessPriority": "LOW"
-        }
-    }
+  "header":{},
+  "payload":{}
 }
 ```
-
-However, for this example, we will include more than the miniumum information. Create a payload file containing the below and name it `addIncident.json`.  
+**Type or copy the following into your Python terminal**  
 ```
-{
-	"payload": {
-		"aptBusinessObjectSummary": "Incomplete shipment received.",
-		"aptBusinessObjectDescription": "Several packages missing or damaged.",
-		"deviationType": "UNPLANNED",
-		"materialType": "FINISHED_GOOD",
-		"materialSubtype": "SHIPPED",
-		"materialProblem": "SHORT_SHIPMENT_RECEIVED",
-		"isEscalated": false,
-		"createdByPartner": false,
-		"isSubmittedToPartner": false,
-		"directSupplierImpact": {
-			"businessImpact": "Business Impact",
-			"businessPriority": "HIGH"
-			}
-		}
+request_body = {
+  "header": {
+    "headerVersion": 1,
+    "eventName": "agile-process-teams:add-direct-supplier-incident:v3",
+    "ownerId": "YOUR_OWNER_ID",
+    "processNetworkId": "YOUR_PROCESS_NETWORK_ID",
+    "appName": "agile-process-teams",
+    "dataspace": "default"
+    },
+  "payload": {
+    "aptBusinessObjectSummary": "Incomplete shipment received.",
+    "aptBusinessObjectDescription": "Several packages missing or damaged.",
+    "deviationType": "UNPLANNED",
+    "materialType": "FINISHED_GOOD",
+    "materialSubtype": "SHIPPED",
+    "materialProblem": "SHORT_SHIPMENT_RECEIVED",
+    "isEscalated": False,
+    "createdByPartner": False,
+    "isSubmittedToPartner": False,
+    "directSupplierImpact": {
+      "businessImpact": "Business Impact",
+      "businessPriority": "HIGH"
+    }
+  }
 }
 ```
+This creates a dictionary object containing the `header` and `payload` elements we discussed earlier.  You can work with the items in this dictionary using standard Python methods.  This allows you to make any desired modifications prior to packaging and submitting your request.  
 
-The `utils.py` file has functions to help build the payload for your request body. The `read_payload_file(filename)` function reads the contents of the file, loads it into a JSON object, and does some minor processing for insertion into the request body.
+As an example, let's change the priority of this incident from `HIGH` to `MEDIUM`.  To change this value, we must reference the `businessPriority` item.  To reach this value, reference the `directSupplierImpact` element of the `payload`.  
```

Comment:
eliminate "let's" and "we" 

For example, change the priority of this incident from `HIGH` to `MEDIUM` by referencing the `businessPriority` item. To reach this value, reference the `directSupplierImpact` element of the `payload`.

Reply:
Updated as suggested